examples/tutorials/CMakeLists.txt

```diff
@@ -1 +1,2 @@
 add_subdirectory(hello_world)
+add_subdirectory(simple_duration)
```

Comment:
I'm not sure if tutorials/ is the best place for it, as we don't have article in the documentation for this feature. You could move it to examples/duration_feature, and once we have an article for it, we can move it back to tutorials